tests/shell/check_resolver.sh

```diff
@@ -190,11 +190,11 @@ check_resolver user b x/a @KDB_DB_HOME@/@KDB_DB_USER@/x/a
 check_resolver user b /a @KDB_DB_HOME@/a
 
 OD=`pwd`
-cd /tmp # hopefully no @KDB_DB_DIR@ is in /tmp
-check_resolver dir b /a /tmp/a
-check_resolver dir b /a/b /tmp/a/b
-check_resolver dir b a /tmp/@KDB_DB_DIR@/a
-check_resolver dir b a/b /tmp/@KDB_DB_DIR@/a/b
+cd $TMPPATH # hopefully no @KDB_DB_DIR@ is in $TMPPATH
```

Comment:
I'm not sure why it is a problem. I just replaced all /tmp occurences with the dynamic one.